vscode-extension/package.json

```diff
@@ -91,6 +123,11 @@
           "type": "boolean",
           "default": true,
           "description": "Enable renaming symbols"
+        },
+        "metals.serverVersion": {
+          "type": "string",
+          "default": "0.1.0",
```

Comment:
How about setting it to the latest published version (`e1b3a1fa`)? Then we can update it once `0.1.0` is out.

Reply:
We could, but I don't intend to publish this until 0.1.0 is out, and I expect metals devs to override this to point to a local snapshot.

So I think it's not an important change now.

Does it make sense?

Reply:
It's up to you. I'd publish it before _and_ after 0.1.0 release 😄

Reply:
Ideally we should be able to publish betas on every commit, but there's no such thing on the Marketplace :( See https://github.com/Microsoft/vscode/issues/15756 and also the lack of API.

So every time we'll be publishing an extension to the store, it will be in correspondence of a new stable release. That's why I think we should default to the latest (in this case the first) stable release.